packages/graphql/src/schema/generation/augment-where-input.ts

```diff
@@ -17,77 +17,245 @@
+                relationshipAdapter.operations.connectionFieldTypename
+            )} match this filter`,
+        },
+        {
+            name: "none",
+            typeName: relationshipAdapter.operations.getConnectionWhereTypename(),
+            description: `Return ${pluralize(relationshipAdapter.source.name)} where none of the related ${pluralize(
+                relationshipAdapter.operations.connectionFieldTypename
+            )} match this filter`,
+        },
+        {
+            name: "single",
+            typeName: relationshipAdapter.operations.getConnectionWhereTypename(),
+            description: `Return ${pluralize(relationshipAdapter.source.name)} where one of the related ${pluralize(
+                relationshipAdapter.operations.connectionFieldTypename
+            )} match this filter`,
+        },
+        {
+            name: "some",
+            typeName: relationshipAdapter.operations.getConnectionWhereTypename(),
+            description: `Return ${pluralize(relationshipAdapter.source.name)} where some of the related ${pluralize(
+                relationshipAdapter.operations.connectionFieldTypename
+            )} match this filter`,
+        },
+    ];
+}
+
+function getRelationshipFiltersLegacy(
+    relationshipAdapter: RelationshipAdapter | RelationshipDeclarationAdapter
+): FieldConfig[] {
+    return [
+        {
+            name: `${relationshipAdapter.name}_ALL`,
+            typeName: relationshipAdapter.target.operations.whereInputTypeName,
+            description: `Return ${pluralize(
+                relationshipAdapter.source.name
+            )} where all of the related ${pluralize(relationshipAdapter.target.name)} match this filter`,
+        },
+        {
+            name: `${relationshipAdapter.name}_NONE`,
+            typeName: relationshipAdapter.target.operations.whereInputTypeName,
+            description: `Return ${pluralize(
+                relationshipAdapter.source.name
+            )} where none of the related ${pluralize(relationshipAdapter.target.name)} match this filter`,
+        },
+        {
+            name: `${relationshipAdapter.name}_SINGLE`,
+            typeName: relationshipAdapter.target.operations.whereInputTypeName,
+            description: `Return ${pluralize(
+                relationshipAdapter.source.name
+            )} where one of the related ${pluralize(relationshipAdapter.target.name)} match this filter`,
+        },
+        {
+            name: `${relationshipAdapter.name}_SOME`,
+            typeName: relationshipAdapter.target.operations.whereInputTypeName,
+            description: `Return ${pluralize(
+                relationshipAdapter.source.name
+            )} where some of the related ${pluralize(relationshipAdapter.target.name)} match this filter`,
+        },
+    ];
+}
+
+function getRelationshipConnectionFiltersLegacy(
+    relationshipAdapter: RelationshipAdapter | RelationshipDeclarationAdapter
+): FieldConfig[] {
+    return [
```

Comment:
These should be deprecated right? Is this a good place to add that deprecation maybe?

Reply:
Absolutely! I'm planning to do in a separate PR!